perception/radar_fusion_to_detected_object/schema/radar_fusion_to_detected_object.schema.json

```diff
@@ -0,0 +1,103 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Radar Fusion to Detected Object Node",
+  "type": "object",
+  "definitions": {
+    "radar_fusion_to_detected_object": {
+      "type": "object",
+      "properties": {
+        "node_params.update_rate_hz": {
```

Comment:
To my knowledge, using "." to create a nested schema isn't possible. In the `joy_controller` node there is an example of how to do this, see:
* https://github.com/autowarefoundation/autoware.universe/pull/3992/files#diff-a7768bad523a7c6d2571183d9240c85ca9829294291c048c8bec15df9b8ff5dfR53

Reply:
Done.